### ISSUE TITLE
Error during custom image building.

### ISSUE THREAD

**Issue by daramayis** [n8n-io/n8n#4361]
**Describe the bug**
The `n8n-custom/Dockerfile` image doesn't build.

**To Reproduce**
1. Go to `git pull origin master`
2. Run `docker build -t n8n-custom -f docker/images/n8n-custom/Dockerfile .`
4. See the error
```
...
Step 4/19 : RUN npm install -g run-script-os turbo
 ---> Running in f22d5ee6e
npm ERR! code 1
npm ERR! path /usr/local/lib/node_modules/turbo
npm ERR! command failed
npm ERR! command sh -c -- node install.js
npm ERR! node:internal/errors:477
npm ERR!     ErrorCaptureStackTrace(err);
npm ERR!     ^
npm ERR! 
npm ERR! <ref *1> Error: spawnSync /usr/local/lib/node_modules/turbo/bin/turbo ENOENT
npm ERR!     at Object.spawnSync (node:internal/child_process:1112:20)
npm ERR!     at spawnSync (node:child_process:827:24)
npm ERR!     at Object.execFileSync (node:child_process:870:15)
npm ERR!     at validateBinaryVersion (/usr/local/lib/node_modules/turbo/install.js:25:6)
npm ERR!     at /usr/local/lib/node_modules/turbo/install.js:308:5 {
npm ERR!   errno: -2,
npm ERR!   code: 'ENOENT',
npm ERR!   syscall: 'spawnSync /usr/local/lib/node_modules/turbo/bin/turbo',
npm ERR!   path: '/usr/local/lib/node_modules/turbo/bin/turbo',
npm ERR!   spawnargs: [ '--version' ],
npm ERR!   error: [Circular *1],
npm ERR!   status: null,
npm ERR!   signal: null,
npm ERR!   output: null,
npm ERR!   pid: 25,
npm ERR!   stdout: null,
npm ERR!   stderr: null
npm ERR! }
```

**Expected behavior**
The image should build without error, and we don't need to fix the version `1.5.5`.

**Comment by netroy** [comment]
Hey @daramayis 
Could you please run this in your n8n directory to ensure that your network doesn't have any issues talking to `npm`?
``docker run -it --rm -v `pwd`:/home/node --entrypoint npm n8nio/base:16 install``

**Comment by daramayis** [comment]
Hey @netroy @Joffcom , looks like my version did not contain the fix https://github.com/n8n-io/n8n/blob/e1e2c943316a29bda989f3d96a7a40e3f57e3bab/docker/images/n8n-custom/Dockerfile#L6

I submitted [PR](https://github.com/n8n-io/n8n/pull/4362), with the required dependency so we will not need to fix the `1.5.5` version.
[(More info here)](https://github.com/vercel/turborepo/issues/2198#issuecomment-1275867758)

**Comment by netroy** [comment]
closed in https://github.com/n8n-io/n8n/pull/4885
